README.md

```diff
@@ -41,6 +41,7 @@ docker-compose -f monitoring/jaeger/docker-compose.yml up
 ```console
 cargo run --release -- daemon \
   --peer-address=127.0.0.1:3000 \
+  --listen-address=0.0.0.0:3001 \
```

Comment:
What does `3001` refer to here?

Reply:
I agree that the "default" of listening on port 3000 which is also the default for Cardano-node is fine, the problem here is in the conflicting situation where we suggest to run locally a Cardano-node on its default port. I think the suggested is fine as far as "getting started" instructions go.

Reply:
Maybe we could suggest to start cardano-node on 3001? This is what we do in the CI.

Reply:
`make bootstrap` relies on a number of environment variables with default values if not set.
The reason `make bootstrap` fails is because the `AMARU_PEER_ADDRESS` is defined as: `AMARU_PEER_ADDRESS ?= 127.0.0.1:3000`.


I think we should just recommend they run the `cardano-node` on a different port and tell them to run:
`$ AMARU_PEER_ADDRESS=[insert address] make bootstrap`. Alternatively, we can default to a public preprod node instead of a local node.

Reply:
Yes this is what I suggested. Also see some context [here](https://github.com/pragma-org/amaru/issues/219).

Reply:
@jeluard yep, sorry wasn't trying to correct you. Just providing additional context to @aleeusgr. I think we should just default to a public preprod node though. It reduces complexity. If it goes down or has issues, we can update it.

Reply:
Sure no worries! It's currently very slow when using a remote public node. I believe we would need to improve `make import-headers` to be usable (to be confirmed). But I agree that this would be the most convenient.

Reply:
If one imports only a couple headers the slowness should be unnoticeable, it becomes much more problematic when importing a whole let of headers, and in this case we'll need to add pipelining to the import logic.

Reply:
Just tried with one of the `play.dev` nodes and while it's ok during `bootstrap` it's very slow and ultimately fails during `sync` (probably due to rate limiting).
My recommandation is to make changes so that the peer node is started on port `3001`.

Reply:
I suppose I can find out how `make bootstrap` works and add an error message there: "make sure cardano-node is running on 3001".

Will do asap🤢🤕🪫

Reply:
@aleeusgr The only change required is to update `AMARU_PEER_ADDRESS`. It requires some agreement first :)

Reply:
I pushed a tentative change making peer default to 3001.